### ISSUE TITLE
Berry bushes block not tagged as `minecraft:crops` in 1.21 Fabric

### ISSUE THREAD

**Issue by TelepathicGrunt** [tallstickman/TallStickMansBetterPies#20]
Can you please tag your crop blocks in `minecraft:crops` block tag? It'll allow bees to grow your crop and allow better mod compat like my mod that will autogrow crops in that tag too.
https://github.com/misode/mcmeta/blob/data-json/data/minecraft/tags/block/crops.json

Specifically it seem `tsm_betterpies:blueberry_bush` and `tsm_betterpies:blackberry_bush` block are an issue

**Comment by tallstickman** [comment]
Yes, I can do that for you.  I will put it I. The todo list.  Thank you for
the suggestion.

TalkStickMan

**Comment by tallstickman** [comment]
Not that it's worth too much over a year later, but I am attempting to resurrect support for this mod.  Therefore, I will attempt to include the resolution for this issue in my next release.  Wish me luck.  :)

**Comment by tallstickman** [comment]
Hmm, it seems I may have promised too much.  My berry bushes are modeled after the Minecraft Sweet Berry Bushes.  In the Java Edition, those are not considered crops.  Incidentally, on Bedrock Edition they are considered crops, but that doesn't help because AFAIK you cannot write a fabric mod for Bedrock.  C++ and all.

That said, I am continuing to look to see if there is a simple way to add my bushes to the crops list anyway, but I may not be able to as easy as I thought.  Thanks again for your suggestion though.

**Comment by TelepathicGrunt** [comment]
`minecraft:bee_growable` may be the proper tag you want then as that contains sweet berry bush block and would work with my mod’s feature.

https://github.com/misode/mcmeta/blob/data-json/data/minecraft/tags/block/bee_growables.json

**Comment by tallstickman** [comment]
Okay, I think I have added the blackberry and blueberry bushes to the minecraft:bee_growables tag.  I have uploaded a version 1.1.4 to Modrinth and marked it as Alpha, since I have not tested it myself yet.  Feel free to try it and let me know if it works for you.

Thanks,

--TSM

**Comment by TelepathicGrunt** [comment]
1.1.4 is for 1.21.4 which my bumblezone mod is not on (latest being 1.21.1 due to client side code breaking changes in mc)

The tag file in your mod looks correct and should be fine

**Comment by tallstickman** [comment]
Great, I'll try and get the change back ported for you then.

**Comment by tallstickman** [comment]
TSM Better Pies version 1.1.2.1 now supports #minecraft:bee_growables.
